pontoon/base/models.py

```diff
@@ -1665,15 +1665,16 @@ class Resource(models.Model):
 
     # Format
     FORMAT_CHOICES = (
-        ('po', 'po'),
-        ('xliff', 'xliff'),
-        ('xlf', 'xliff'),
-        ('properties', 'properties'),
         ('dtd', 'dtd'),
+        ('ftl', 'ftl'),
         ('inc', 'inc'),
         ('ini', 'ini'),
+        ('json', 'json'),
```

Comment:
`.json` is the general format for various types of data. I'm not sure how the layout of the future repositories will look like.
Maybe we should restrict our parsers only to  parse only files named `messages.json` and additionaly use http://json-schema.org/ to validate if the schema is correct?

Reply:
ive been using json-schema and it rocks - seems like a good way to validate files are actually messages

Reply:
Thanks, I'll look into that.